homeassistant/components/tplink/light.py

```diff
@@ -177,49 +180,59 @@ def is_on(self):
 
-            if self._supported_features & SUPPORT_COLOR_TEMP:
-                if (
-                    self.smartbulb.color_temp is not None
-                    and self.smartbulb.color_temp != 0
-                ):
-                    self._color_temp = kelvin_to_mired(self.smartbulb.color_temp)
+                if self._supported_features & SUPPORT_COLOR_TEMP:
+                    if (
+                        self.smartbulb.color_temp is not None
+                        and self.smartbulb.color_temp != 0
+                    ):
+                        self._color_temp = kelvin_to_mired(self.smartbulb.color_temp)
 
-            if self._supported_features & SUPPORT_COLOR:
-                hue, sat, _ = self.smartbulb.hsv
-                self._hs = (hue, sat)
+                if self._supported_features & SUPPORT_COLOR:
+                    hue, sat, _ = self.smartbulb.hsv
+                    self._hs = (hue, sat)
 
-            if self.smartbulb.has_emeter:
-                self._emeter_params[ATTR_CURRENT_POWER_W] = "{:.1f}".format(
-                    self.smartbulb.current_consumption()
-                )
-                daily_statistics = self.smartbulb.get_emeter_daily()
-                monthly_statistics = self.smartbulb.get_emeter_monthly()
-                try:
-                    self._emeter_params[ATTR_DAILY_ENERGY_KWH] = "{:.3f}".format(
-                        daily_statistics[int(time.strftime("%d"))]
-                    )
-                    self._emeter_params[ATTR_MONTHLY_ENERGY_KWH] = "{:.3f}".format(
-                        monthly_statistics[int(time.strftime("%m"))]
+                if self.smartbulb.has_emeter:
+                    self._emeter_params[ATTR_CURRENT_POWER_W] = "{:.1f}".format(
+                        self.smartbulb.current_consumption()
                     )
-                except KeyError:
-                    # device returned no daily/monthly history
-                    pass
-
-            self._available = True
-
-        except (SmartDeviceException, OSError) as ex:
+                    daily_statistics = self.smartbulb.get_emeter_daily()
+                    monthly_statistics = self.smartbulb.get_emeter_monthly()
+                    try:
+                        self._emeter_params[ATTR_DAILY_ENERGY_KWH] = "{:.3f}".format(
+                            daily_statistics[int(time.strftime("%d"))]
+                        )
+                        self._emeter_params[ATTR_MONTHLY_ENERGY_KWH] = "{:.3f}".format(
+                            monthly_statistics[int(time.strftime("%m"))]
+                        )
+                    except KeyError:
+                        # device returned no daily/monthly history
+                        pass
+
+                self._available = True
+
+            except (SmartDeviceException, OSError) as ex:
+                _LOGGER.warning(
+                    f"Retrying in {SLEEP_TIME} for {self.smartbulb.host}|{self._alias} due to: {ex}"
```

Comment:
Please revert the logging formatting changes. We use standard logging formatting.